python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1448,11 +1461,14 @@ def run(self, visitSummaryRefs):
             # Technically you should join to get the visit from the visit
             # table.
             ccdEntry = ccdEntry.rename(columns={"visit": "visitId"})
-            dataIds = [DataCoordinate.standardize(visitSummaryRef.dataId, detector=id) for id in
-                       summaryTable['id']]
-            packer = visitSummaryRef.dataId.universe.makePacker('visit_detector', visitSummaryRef.dataId)
-            ccdVisitIds = [packer.pack(dataId) for dataId in dataIds]
-            ccdEntry['ccdVisitId'] = ccdVisitIds
+            ccdEntry['ccdVisitId'] = [
+                self.config.idGenerator.apply(
+                    visitSummaryRef.dataId,
+                    detector=detector_id,
+                    is_exposure=False,
+                ).catalog_id
```

Comment:
Assigning a "catalog_id" to a "ccdVisitId" feels a bit odd to me. Should we rename the "ccdVisitId" database key (which is why we have this in the first place), or just live with this slightly confusing assignment?

Reply:
I'll add a comment; this is a slightly unusual usage of `IdGenerator` and that's why it seems strange.  It's a valid usage but a rare enough one that I don't that to inspire any name changing in `IdGenerator` itself.